pyproject.toml

```diff
@@ -11,7 +11,7 @@ python = "~3.10"
 pytorch = "2.0.1"
 pytorch-cuda = "11.8"
 deepspeed = "0.12.2"
-transformers = "4.35.0"
+transformers = "4.36.0"
```

Comment:
The lock file also needs to be updated.

Reply:
ok, i will update it and test

Reply:
done and test passed:
![image](https://github.com/01-ai/Yi/assets/66633207/ca11e7dd-74db-404f-ab3f-970c550f14b0)